conans/test/unittests/util/client_conf_test.py

```diff
@@ -49,3 +62,74 @@ def test_proxies(self):
         save(os.path.join(tmp_dir, CONAN_CONF), "[proxies]\nno_proxy=localhost")
         config = ConanClientConfigParser(os.path.join(tmp_dir, CONAN_CONF))
         self.assertEqual(config.proxies["no_proxy"], "localhost")
+
+
+class ClientConfLogTest(unittest.TestCase):
+
+    def setUp(self):
+        self.tmp_dir = temp_folder()
+        save(os.path.join(self.tmp_dir, DEFAULT_PROFILE_NAME), default_profile)
+        try:
+            del os.environ["CONAN_LOGGING_LEVEL"]
+        except:
+            pass
+
+    def test_log_level_numbers_critical(self):
+        save(os.path.join(self.tmp_dir, CONAN_CONF), default_client_conf_log.format("level = 50"))
+        config = ConanClientConfigParser(os.path.join(self.tmp_dir, CONAN_CONF))
+        self.assertEqual(logging.CRITICAL, config.logging_level)
+
+    def test_log_level_numbers_debug(self):
+        save(os.path.join(self.tmp_dir, CONAN_CONF), default_client_conf_log.format("level = 10"))
+        config = ConanClientConfigParser(os.path.join(self.tmp_dir, CONAN_CONF))
+        self.assertEqual(logging.DEBUG, config.logging_level)
+
+    def test_log_level_numbers_invalid(self):
+        save(os.path.join(self.tmp_dir, CONAN_CONF), default_client_conf_log.format("level = wakawaka"))
```

Comment:
Wow. Shouldn't we rise?

Reply:
from early condition, any error is treated as default logging level (CRITICAL). Should we change it?

Reply:
The condition is here: https://github.com/conan-io/conan/blob/1.18.5/conans/client/conf/__init__.py#L511